recipes/mosquitto/mosquitto_1.4.10.oe

```diff
@@ -0,0 +1,3 @@
+require ${PN}.inc
+LIBRARY_VERSION = "1"
+SRC_URI += "file://0001-dont_strip.patch"
```

Comment:
right, i don't see the commit msg when viewing all the changes at once. Would you mind adding a patch header here like we have started doing other places:
`#source: where is the patch originating from`
`#reason: why is the patch there`
`#upstream: has it been submitted upstream?`
makes it much easier when we want to upgrade it next time.